### ISSUE TITLE
What parameters should I change to achieve a better mIoU when training spconv in a photogrammetric dataset?

### ISSUE THREAD

**Issue by LuZaiJiaoXiaL** [Pointcept/Pointcept#10]
Hi, may I know what parameters in the config file or in your code are sensitive to different datasets?

Currently, I have a photogrammetric dataset capturing city objects at the urban scale. I want to modify the parameters that you provided for S3DIS to train the spconv model. But I'm not sure what parameters I shall finetune, and to what extent should I change?

Your suggestions would be much appreciated.

Cheers,

Eric

**Comment by Gofinge** [comment]
Hi, here are my suggestions:

1. Tuning voxel size in `Voxelization` to make sure the input scale is fit your GPU memory;
2. Tuning learning rate;
3. Turning other augmentation;

**Comment by Gofinge** [comment]
Yes, initial grid sampling, similar to resolution of image.
